packages/dart/npt_flutter/lib/features/onboarding/util/atsign_manager.dart

```diff
@@ -19,12 +44,119 @@ abstract class AtsignInformation {
 // Now you have the rootDomain for the existing atSign and can use it to onboard
 // correctly
 
-Future<Map<String, AtsignInformation>> getAtsignEntries() {
-  return Future.value({});
+Future<Map<String, AtsignInformation>> getAtsignEntries() async {
```

Comment:
Use this method to determine which atSigns may be re-onboarded from the keyvhain.
Inside the AtsignInformation, you will find the rootDomain, which should be set before onboarding the atSign (contained inside the AtSignInformation, and the key of the map)